GameData/ROCapsules/PartConfigs/Apollo/ApolloCM.cfg

```diff
@@ -1,654 +1,655 @@
 PART
```

Comment:
Did this entire file just change tabs and spaces?  I cannot tell what is different in here.

Reply:
Just changed to tabs

Reply:
What changed in this file.  I cannot evaluate 600 lines of addition and 600 lines of deletion to search for anything that -wasn't- a whitespace change.